package.json

```diff
@@ -1,45 +1,132 @@
 {
   "name": "vanilla-framework-react",
-  "version": "0.1.0",
-  "private": true,
-  "homepage": "https://vanilla-framework.github.io/vanilla-components/",
-  "dependencies": {
-    "node-sass-chokidar": "0.0.3",
-    "npm-run-all": "^4.1.1",
-    "prop-types": "^15.6.0",
-    "raf": "^3.4.0",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
-    "react-scripts": "1.0.14",
-    "@storybook/react": "^3.2.13",
-    "@storybook/addon-actions": "^3.2.13",
-    "@storybook/addon-links": "^3.2.13",
-    "@storybook/addon-info": "^3.2.14"
-  },
+  "version": "0.0.5",
```

Comment:
Dropping the version from `0.1.0` to `0.0.5`?

Reply:
I think this is fine, that is the current version on NPM and `0.0.x` may be a better indication of the project.